scripts/mounts.env

```diff
@@ -28,21 +28,21 @@
 #   data_set - the z/OS data set containing the binaries to mount
 #   space - must be a space before the closing quote
 # ------------------------------------------------------------------------------
-zoau_mount_list_str="1:1.2.0:/zoau/v1.2.0:IMSTESTU.ZOAU.V120.ZFS "\
-"2:1.0.0-ga:/zoau/v1.0.0-ga:IMSTESTU.ZOAU.V100.GA.ZFS "\
-"3:1.0.1-ga:/zoau/v1.0.1-ga:IMSTESTU.ZOAU.V101.GA.ZFS "\
-"6:1.0.2-ga:/zoau/v1.0.2-ga:IMSTESTU.ZOAU.V102.GA.ZFS "\
-"7:1.0.3-ga5:/zoau/v1.0.3-ga5:IMSTESTU.ZOAU.V103.GA5.ZFS "\
-"8:1.0.3-ptf2:/zoau/v1.0.3-ptf2:IMSTESTU.ZOAU.V103.PTF2.ZFS "\
-"9:1.1.0-ga:/zoau/v1.1.0-ga:IMSTESTU.ZOAU.V110.GA.ZFS "\
-"10:1.1.1-ptf1:/zoau/v1.1.1-ptf1:IMSTESTU.ZOAU.V111.PTF1.ZFS "\
-"11:1.2.1:/zoau/v1.2.1:IMSTESTU.ZOAU.V121.ZFS "\
-"12:1.2.2:/zoau/v1.2.2:IMSTESTU.ZOAU.V122.ZFS "\
-"13:1.2.3:/zoau/v1.2.3:IMSTESTU.ZOAU.V123.ZFS "\
-"14:1.2.4:/zoau/v1.2.4:IMSTESTU.ZOAU.V124.ZFS "\
-"15:1.2.5:/zoau/v1.2.5:IMSTESTU.ZOAU.V125.ZFS "\
-"16:1.3.0:/zoau/v1.3.0:IMSTESTU.ZOAU.V130.ZFS "\
-"17:latest:/zoau/latest:IMSTESTU.ZOAU.LATEST.ZFS "
+zoau_mount_list_str="1:1.0.0-ga:/zoau/v1.0.0-ga:IMSTESTU.ZOAU.V100.GA.ZFS "\
+"2:1.0.1-ga:/zoau/v1.0.1-ga:IMSTESTU.ZOAU.V101.GA.ZFS "\
+"3:1.0.2-ga:/zoau/v1.0.2-ga:IMSTESTU.ZOAU.V102.GA.ZFS "\
```

Comment:
ZOAU.V102.GA.ZFS is now used for 1.2.5.8, 'latest' is a misnomer, and relates to 1.2.5.6, but I haven't changed the markings yet

Reply:
Thanks @richp405 this line was supposed to have been removed, I had it below as part of 1.2.5, `"13:1.2.5:/zoau/v1.2.5:IMSTESTU.ZOAU.V102.GA.ZFS "\`, I will update this.

Reply:
Updated, thanks @richp405